src/reimer_arenkiel_lab_to_nwb/dj_utils.py

```diff
@@ -193,73 +180,52 @@ def add_summary_images(nwbfile: NWBFile, key: dict = None, verbose: bool = False
     ),
     Ophys=dict(
         Device=dict(
-            name="Two-photon microscope",
-            description="My two-photon microscope",
-            # manufacturer="The best microscope manufacturer",
+            name="two_photon_microscope",
+            description="Janelia 2P-RAM mesoscope.",
+            manufacturer="ThorLabs",
         ),
         OpticalChannel=dict(
-            name="OpticalChannel",
-            description="an optical channel",
-            emission_lambda=500.0,
+            name="green_channel",
+            description="Emitted light was collected through a 525/50 filter and a gallium arsenide phosphide "
+                        "photomultiplier tube (Hamamatsu).",
+            emission_lambda=525.0,
         ),
         ImagingPlane=dict(
-            imaging_rate=30.0,
-            excitation_lambda=600.0,
-            indicator="GFP",
-            location="V1",
-            # grid_spacing=[0.01, 0.01],
-            # grid_spacing_unit="meters",
-            # origin_coords=[1.0, 2.0, 3.0],
-            # origin_coords_unit="meters",
+            name="imaging_plane",
+            description="Imaging plane for the Green channel recorded with 2p microscope.",
+            excitation_lambda=920.0,
+            indicator="GCaMP6f",
+            location="Whole Brain",
         ),
     )
 )
 
 
 def add_imaging_plane(
-    nwbfile: NWBFile, device: Device, key: dict = None, metadata: dict = None, verbose: bool = False
+        nwbfile: NWBFile, device: Device, key: dict = None, metadata: dict = None, verbose: bool = False
```

Comment:
why not using `add_imaging_plane` from neuroconv?

Reply:
I am not sure, this fun was added by Ben. But I guess we could use the one from neuroconv